tests/functional/workloads/app/amq/test_rgw_kafka_notifications.py

```
@@ -186,20 +216,37 @@ def test_rgw_kafka_notifications(self, rgw_bucket_factory):
+                    f"Timestamp event {event_time} matches the pattern {format_string}"
+                )
             except ValueError as ef:
-                log.error(
-                    f"Timestamp event {event_time} doesnt match the pattern {format_string}"
+                logger.assertion(
+                    f"Timestamp validation: event_time='{event_time}', "
+                    f"format='{format_string}', valid=False"
+                )
+                logger.error(
+                    f"Timestamp event {event_time} doesn't match the pattern {format_string}"
                 )
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)